contracts/conduit/ConduitController.sol

```diff
@@ -198,8 +213,13 @@ contract ConduitController is ConduitControllerInterface {
             revert NewPotentialOwnerIsZeroAddress(conduit);
         }
 
+        // Ensure the new potential owner is not already set.
+        if (newPotentialOwner == _conduits[conduit].potentialOwner) {
```

Comment:
This only checks for the same owner being proposed twice, but allows proposing a new one while a proposal is active. Is that the goal?

Reply:
I think that’s fine, the current owner is still in control at that stage

Reply:
It does feel like a not very useful check though.